src/components/AppRouter/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Suspense } from "react";
+import { Routes } from "react-router";
+import { Route } from "react-router-dom";
+import { RouteConfig } from "shared/config";
```

Comment:
This configuration is related to the AppRouter only.
Move from the global level into AppRouter.